src/cs/validation.php

```diff
@@ -34,15 +34,15 @@
     'different'            => ':attribute a :other se musí lišit.',
     'digits'               => ':attribute musí být :digits pozic dlouhé.',
     'digits_between'       => ':attribute musí být dlouhé nejméně :min a nejvíce :max pozic.',
-    'dimensions'           => 'The :attribute has invalid image dimensions.',
-    'distinct'             => 'The :attribute field has a duplicate value.',
+    'dimensions'           => ':attribute má neplatné rozměry.',
+    'distinct'             => ':attribute má duplicitní hodnotu.',
     'email'                => ':attribute není platný formát.',
     'exists'               => 'Zvolená hodnota pro :attribute není platná.',
     'file'                 => 'The :attribute must be a file.',
```

Comment:
Fixed in https://github.com/caouecs/Laravel-lang/pull/572 - didn't notice this before, sorry.